content/en/getting_started/session_replay/_index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Getting Started with Session Replay
+kind: documentation
+further_reading:
+    - link: 'https://docs.datadoghq.com/real_user_monitoring/session_replay/'
+      tag: 'Documentation'
+      text: 'Session Replay'
+    - link: 'https://docs.datadoghq.com/real_user_monitoring/session_replay/mobile/'
+      tag: 'Documentation'
+      text: 'Mobile Session Replay'
+    - link: 'https://www.datadoghq.com/knowledge-center/session-replay/'
+      tag: 'Learning Center'
+      text: 'Session Replay Overview'
+    - link: 'https://www.datadoghq.com/blog/session-replay-datadog/'
+      tag: 'Blog'
+      text: 'Use Datadog Session Replay to view real-time user journeys'
+    - link: 'https://docs.datadoghq.com/real_user_monitoring/browser/troubleshooting/'
+      tag: 'Documentation'
+      text: 'Troubleshooting'
+---
+
+## Overview
+
+**Session Replay** is a visual tool that recreates user sessions from your applications, giving you an in-depth look at how users are interacting with your product. Session Replay can be helpful in understanding an application's pain points, as well as other nuances that may be harder to detect when looking at a session timeline, but easier to notice in a visual representation. 
+
+This page demonstrates how to get started with Session Replay in Datadog. If you haven't already, [create a Datadog account][1].
+
+## Configuring Session Replays
+
+Session Replay is available in the RUM Browser SDK. To start collecting data for Session Replay, set up Datadog RUM Browser Monitoring by creating a RUM application, generating a client token generation, and initializing the RUM Browser SDK. 
+
+For further instructions on collecting Session Replay data, follow the [RUM Getting Started guide][2] for your application. 
+
+For setup in mobile environments, see [Mobile Session Replay][3].
+
+## Finding particular Session Replays
+
+Once you're collecting Session Replay data, select **Session Replay available** in the [RUM Explorer][4] to see all sessions with a replay attached to it. From here, you can visualize the data as a **List**, **Timeseries**, or other format. 
+
+{{< img src="/getting_started/session_replay/session-replay-available.png" alt="Session Replay available button, as well as visualization options">}}
+
+### Facets
+
+You can filter Sessions using facets. Filtering by [facet][5] is helpful if you are searching for specific information, such as a particular user or device type. 
+
+### Views
+
+You are also able to filter by [view][6]. This can be helpful in the case that you know where the issue lies.
```

Comment:
I'd recommend adding an image that captures the Views dropdown menu or the My View button like: 
![Screenshot 2023-09-12 at 8 34 19 AM](https://github.com/DataDog/documentation/assets/76412946/b5528191-7ac6-421b-adbf-fe72d01ae91a)